ownCloud/Client/ClientQueryViewController.swift

```diff
@@ -269,26 +270,22 @@ class ClientQueryViewController: UITableViewController, Themeable {
 	}
 
 	override func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
-
-		guard let rowItem : OCItem = self.items?[indexPath.row] else {
-			return
-		}
+		let rowItem : OCItem = self.items![indexPath.row]
```

Comment:
Maybe out of the scope of this PR but we've removed the 'optionality' of the `core` and `query` in this class, we could do the same with the `items` property. This way we could avoid all the nil checking and the `!` `?` symbols.

Reply:
Made that change in the latest commit.